chrome-extension/src/utils/ExecuteActionsUtils.ts

```diff
@@ -0,0 +1,68 @@
+import { ActionsEnum, StepsResponseType, StepsType } from "../types/steps";
+
+export const sendSteps = async (steps: StepsType[]) => {
+  for await (const step of steps) {
+    const image = await chrome.tabs.captureVisibleTab();
+    const data: StepsType = {
+      instruction: {
+        description: step.instruction.description,
+        value: step.instruction.value,
+        action: step.instruction.action,
+      },
+      currentView: image,
+      index: step.index,
+    };
+
+    try {
+      // send the data to the backend
+      const call = await fetch("http://localhost:3000/steps", {
```

Comment:
pasa esta wea como env var
```suggestion
      const call = await fetch(`${API_URL}/steps`, {
```